### DISCUSSION TITLE
share form schema between resources with some modification

### DISCUSSION THREAD

**Discussion by abanghendri** [filamentphp/filament#12158] [Help]
### Package

Form builder

### Package Version

v.3.2.47

### How can we help you?

Hi, I have 2 resources, PostResources and PageResources, since both use same model:Content I try to share form schema from PostResources to PageResources using this method
```
public static function form(Form $form): Form
    {
        return PostsResource::form($form);
    }
```
my Question is, I don't want to show tag and category fields below inside PageResources
```
 Section::make('Tags and Category')->schema([
                            Select::make('category_id')->relationship('category', 'title')->preload(),
                            TagsInput::make('tags')
                            ->splitKeys([','])
                            ->helperText(new HtmlString('press <kbd class="kbd">,</kbd> or <kbd class="kbd">Enter</kbd> to save tag')),
                        ])->collapsible(),
```
How to achieve this?
I've found the documentation mentioning about utility injection but none of the function telling us how to read in which resource we call the form, do you have any solution for this?

**Comment by micraux** [comment]
[https://filamentphp.com/docs/3.x/panels/resources/relation-managers#hiding-a-shared-form-component-on-the-relation-manager](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#hiding-a-shared-form-component-on-the-relation-manager) ?
